elyra/pipeline/component_parameter.py

```diff
@@ -378,7 +457,10 @@ def get_schema(cls) -> Dict[str, Any]:
         return schema
 
     def get_value_for_dict_entry(self) -> str:
```

Comment:
We should also update the return value types of each override to that of the superclass: `Union[str, Dict[str, Any]]:`